setup.py

```diff
@@ -313,15 +313,29 @@ def get_nccl_dirs(build_ext, cuda_include_dirs, cuda_lib_dirs, cpp_flags):
 
     return nccl_include_dirs, nccl_lib_dirs
 
+def get_ddl_dirs():
+    # Default DDL home
+    ddl_home = '/opt/DL/ddl'
```

Comment:
We should make this location configurable.

Reply:
Yes we could. But It will most likely never change because our rpm and deb packages assume it gets setup to this location.

Reply:
Is it always rpm and deb, or you also have tarball with the libraries?

Reply:
Yes always rpm or dep.

Reply:
OK, it's fine to keep like that then.